packages/editor/src/components/post-title/index.js

```diff
@@ -123,7 +123,7 @@ class PostTitle extends Component {
 							<Textarea
 								id={ `post-title-${ instanceId }` }
 								className="editor-post-title__input"
-								value={ unescape( title ) }
+								value={ decodeEntities( title ) }
```

Comment:
What are the differences in implementation? Can we add a unit test to catch the broken case here?

Reply:
Some level of filtering on wpcom converts  `#39;` to `#039;`. lodash `unescape` only looks for an exact match of `#39;`. WP `decodeEntities` on the other hand, doesn't search for specific entity strings, it just throws everything into a temporary text field in the dom and then extracts, so relies entirely on the browsers own encoding/decoding, which handles both `#39;` and `#039;`

Reply:
Added an additional test to cover this variation in entity codes.